src/utils.cpp

```diff
@@ -0,0 +1,20 @@
+#include "utils.hpp"
+
+inline constexpr uint8_t BITS_IN_BYTE = 8;
+
+template <class UnsignedInteger>
+    requires std::unsigned_integral<UnsignedInteger>
+std::vector<uint8_t> utils::intToBytes(UnsignedInteger value)
+{
+    // This cast is safe, the biggest we can get is 8.
```

Comment:
Are you sure about that?
realPduDataFieldLength is uint16

Reply:
yeah, we can call it on u8 to u64